django_project/settings/base.py

```diff
@@ -240,7 +240,7 @@
     },
     'update_ns_propagated': {
         'task': 'zinc.tasks.update_ns_propagated',
-        'schedule': 60
+        'schedule': 60 * 10,
```

Comment:
Why's that?

Reply:
The NS propagated task checks if the actual NS used by us are set or not. This helps in fast debuging, it's not an important task, based on this info there are not taken any decisisons.
So I increased it to 10 minutes.

I will use `timedelta`.